### PR TITLE
Minor Telegram and UpdateLiveStats fixes 

### PR DESCRIPTION
**Pull request by DBa2016**
## Short Description:
- Minor change to the "/help" output on Telegram
- UpdateLiveStats has now "stardust_amount" - to display overall stardust amount.

### CODE REVIEW

**Comment by DBa2016** [comment]
Hmmm... Why did Travis fail?

**Comment by alexyaoyang** [comment]
These two tests failed:

```
======================================================================
ERROR: test_get_stats_line (tests.update_live_stats_test.UpdateLiveStatsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/tests/update_live_stats_test.py", line 177, in test_get_stats_line
    line = self.worker._get_stats_line(self.worker._get_stats(self.player_stats))
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/pokemongo_bot/cell_workers/update_live_stats.py", line 266, in _get_stats
    stardust_amount = self.bot.stardust
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/pokemongo_bot/__init__.py", line 72, in stardust
    return filter(lambda y: y['name'] == 'STARDUST', self._player['currencies'])[0]['amount']
KeyError: u'currencies'
======================================================================
ERROR: test_get_stats_line_no_displayed_stats (tests.update_live_stats_test.UpdateLiveStatsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/tests/update_live_stats_test.py", line 170, in test_get_stats_line_no_displayed_stats
    line = self.worker._get_stats_line(self.worker._get_stats(self.player_stats))
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/pokemongo_bot/cell_workers/update_live_stats.py", line 266, in _get_stats
    stardust_amount = self.bot.stardust
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/pokemongo_bot/__init__.py", line 72, in stardust
    return filter(lambda y: y['name'] == 'STARDUST', self._player['currencies'])[0]['amount']
KeyError: u'currencies'
```

File: pokemongo_bot/cell_workers/update_live_stats.py [hunk 1]
```diff
@@ -344,6 +343,7 @@ def _get_stats_line(self, player_stats):
             ),
             'pokeballs_thrown': 'Threw {:,} pokeballs'.format(player_stats['pokeballs_thrown']),
             'stardust_earned': 'Earned {:,} Stardust'.format(player_stats['stardust_earned']),
+            'stardust_amount': 'Have {:,} Stardust'.format(player_stats['stardust_amount']),
```
Comment (Gobberwart) [inline pokemongo_bot/cell_workers/update_live_stats.py]: 'stardust_amount' is not in player_stats. Use self.bot.stardust instead, or metrics.dust.latest.

**Comment by cmezh** [comment]
Travis still fails.
```
======================================================================
ERROR: test_get_stats_line (tests.update_live_stats_test.UpdateLiveStatsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/tests/update_live_stats_test.py", line 177, in test_get_stats_line
    line = self.worker._get_stats_line(self.worker._get_stats(self.player_stats))
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/pokemongo_bot/cell_workers/update_live_stats.py", line 346, in _get_stats_line
    'stardust_amount': 'Have {:,} Stardust'.format(self.bot.metrics.dust['latest']),
ValueError: Cannot specify ',' with 's'.
======================================================================
ERROR: test_get_stats_line_no_displayed_stats (tests.update_live_stats_test.UpdateLiveStatsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/tests/update_live_stats_test.py", line 170, in test_get_stats_line_no_displayed_stats
    line = self.worker._get_stats_line(self.worker._get_stats(self.player_stats))
  File "/home/travis/build/PokemonGoF/PokemonGo-Bot/pokemongo_bot/cell_workers/update_live_stats.py", line 346, in _get_stats_line
    'stardust_amount': 'Have {:,} Stardust'.format(self.bot.metrics.dust['latest']),
ValueError: Cannot specify ',' with 's'.
```

File: pokemongo_bot/cell_workers/update_live_stats.py [hunk 2]
```diff
@@ -344,6 +343,7 @@ def _get_stats_line(self, player_stats):
             ),
             'pokeballs_thrown': 'Threw {:,} pokeballs'.format(player_stats['pokeballs_thrown']),
             'stardust_earned': 'Earned {:,} Stardust'.format(player_stats['stardust_earned']),
+            'stardust_amount': 'Have {:,} Stardust'.format(self.bot.metrics.dust['latest']),
```
Comment (rawgni) [inline pokemongo_bot/cell_workers/update_live_stats.py]: try casting this to int or just get rid of `:,`

**Comment by rawgni** [comment]
re: test_get_stats_line_no_displayed_stats

The _get_stat code used to look like this
```
    def _get_stats_line(self, player_stats):
        """
        Generates a stats string with the given player stats according to the configuration.
        :return: A string containing human-readable stats, ready to be displayed.
        :rtype: string
        """
        # No player stats available, won't be able to gather all informations.
        if player_stats is None:
            return ''
        # No stats to display, avoid any useless overhead.
        if not self.displayed_stats:
            return ''
```

note the `if not self.displaye_stats` statement, it's now missing :(

File: pokemongo_bot/cell_workers/update_live_stats.py (hunk 2 above)
Comment (Gobberwart) [inline pokemongo_bot/cell_workers/update_live_stats.py]: yeah metrics.dust['latest'] appears to just call bot.stardust which is a bit dodgy

**Comment by DBa2016** [comment]
Wohoo, Travis finally happy! :)

**Comment by solderzzc** [comment]
> This branch has conflicts that must be resolved

**Comment by DBa2016** [comment]
I suggest reverting #5371, which introduced these conflicts and needs more testing anyway.

**Comment by javajohnHub** [comment]
I disagree that #5371 caused any issue.

**Comment by solderzzc** [comment]
@javajohnHub  agree.

**Pull request closed without merging**